Gemfile.lock

```diff
@@ -75,12 +81,41 @@ GEM
       simplecov_json_formatter (~> 0.1)
     simplecov-html (0.12.3)
     simplecov_json_formatter (0.1.4)
-    sorbet (0.5.10365)
-      sorbet-static (= 0.5.10365)
-    sorbet-runtime (0.5.10365)
-    sorbet-static (0.5.10365-x86_64-linux)
+    sorbet (0.5.10346)
```

Comment:
I tried for quite a bit but couldn't get sorbet to be bumped given the new `Gemfile.lock`. Nothing jumps out to me that is limiting Sorbet version 😕 Maybe you can have more luck locally.

Reply:
That's likely because there are no darwin builds for the new release due to https://sorbet-ruby.slack.com/archives/CHN2L03NH/p1660847337669779

Reply:
Good find, I was very confused.